src/utils/parseConfigStrings.ts

```diff
@@ -0,0 +1,29 @@
+export function parseConfigStrings(configurations?: string[]): any {
+  if (!configurations) return
+
+  const config: any = {}
+
+  const configuration = configurations.join(";").trim()
+
+  const configFields = configuration.split(";")
```

Comment:
What? join first and split again immediately afterwards?

Reply:
This makes `["a=b;c=d", "e=f"]` => `"a=b;c=d;e=f"` => `["a=b", "c=d", "e=f"]`

I don't like the other solutions.

Reply:
```ts
["a=b;c=d", "e=f"].map((config) => config.trim().split(";")).flat()
// => [ 'a=b', 'c=d', 'e=f' ]
```

Reply:
flatMap is even better. changed.